frontend/src/components/admin/path-connection/list-item/styles.scss

```diff
@@ -0,0 +1,16 @@
+@import 'assets/scss/utils';
+
```

Comment:
this css file and [frontend/src/components/admin/path-connection/levels-list/styles.scss](url) are identical,
you should use only one css file